x/wasm/keeper/keeper_test.go

```diff
@@ -1042,9 +1042,16 @@ func TestMigrateWithDispatchedMessage(t *testing.T) {
 				{"module": "bank"},
 			},
 		},
+		{
```

Comment:
The events look mostly right, but the ordering all wrong.

I would expect 

1. no `message` events (they should be dropped inside the keeper, the module: wasm one added in msg server
2. first event is migrate
3. then wasm
4. then transfer

basically, placing migrate on top and filtering out the messages and this looks better to me. let me look at the keeper code again